go/mysql/query_test.go

```diff
@@ -454,8 +459,13 @@ func checkQueryInternal(t *testing.T, query string, sConn, cConn *Conn, result *
 	}
 
 	wg.Wait()
+
+	if fatalError != "" {
+		t.Fatalf(fatalError)
+	}
 }
 
+//lint:ignore U1000 for now, because deleting this function causes more errors
```

Comment:
Is there a bug in the linter?

Reply:
Nope, not a bug. Removing this function makes some of the others it is calling show up as unused. I don't know if there is a reason to keep them around.

Reply:
Decided to keep the function in case we want to use it in tests.